src/pybind/mgr/cephadm/inventory.py

```diff
@@ -1699,6 +1764,294 @@ def agent_config_successfully_delivered(self, daemon_spec: CephadmDaemonDeploySp
         self.save_agent(daemon_spec.host)
 
 
+class Cert():
+    def __init__(self, cert: str = '', user_made: bool = False) -> None:
+        self.cert = cert
+        self.user_made = user_made
```

Comment:
Do we really need this field? we can inspect the cert to see if it was self-generated or not? is it to make this check faster?

Reply:
Since we can tell when the cert is being stored if it's user made or not and it's inexpensive to store that info, I thought we might as well try to track it. I like having it tracked explicitly rather than trying to figure it out whenever we may need it. That's sort of a general theme of how this cert store is being built with the types of certs we store being explicitly listed.

Reply:
I see, I'm OK with keeping the `user_made` boolean :+1: